symbols/n_jeft_transistor_horz.ts

```diff
@@ -0,0 +1,20 @@
+import { defineSymbol } from "drawing/defineSymbol"
+import svgJson from "assets/generated/n_jeft_transistor.json"
+
+const { paths, texts, bounds, refblocks, circles } = svgJson
+
+export default defineSymbol({
+  primitives: [
+    ...Object.values(paths),
+    ...Object.values(circles),
+    { ...texts.top1, anchor: "middle_center" },
+    { ...texts.bottom1, anchor: "middle_center", x:-0.15},
+  ] as any,
+  ports: [
+    { ...refblocks.top1, labels: ["1"] }, // TODO add more "standard" labels
+    { ...refblocks.bottom1, labels: ["2"] }, // TODO add more "standard" labels
+    { ...refblocks.left1, labels: ["3"] }, // TODO add more "standard" labels
```

Comment:
can you add "gate", "source" and "drain" here

Reply:
done